layouts/default.vue

```diff
@@ -29,6 +29,7 @@ export default {
     return {
       meta: [
         { hid: "og:url", property: "og:url", content: SITE_URL + this.$route.path },
+        { hid: "theme-color", color: "#292929" },
```

Comment:
@frederickobrien We actually already have one of these here:

https://github.com/audioxide/website/blob/18233c3c31999dc9b474a09dffadb6c2763a7743/nuxt.config.js#L67

Did Lighthouse complain that it didn't exist?

Reply:
Huh, that's odd. It was flagged as an issue here: https://lighthouse-metrics.com, @andrewbridge. Could be a problem with the tool?

Reply:
Just checked this, I think it isn't complaining about the `<meta>`. From the help page:

> The audit fails if Lighthouse doesn't find a theme-color meta tag in the page's HTML **and a `theme_color` property in the web app manifest.**

Emphasis is mine, I think we need to add the value to our web app manifest. This is something that is at least partially managed by Nuxt, so we might need to dig into their documentation to work out how to do this correctly.